nomad/structs/job.go

```diff
@@ -132,3 +132,15 @@ func (j *Job) RequiredNUMA() set.Collection[string] {
 	}
 	return result
 }
+
+// RequiredBridgeNetwork identifies which task groups, if any, within the job
+// contain networks requesting bridge networking.
+func (j *Job) RequiredBridgeNetwork() set.Collection[string] {
+	result := set.New[string](10)
```

Comment:
```suggestion
	result := set.New[string](len(j.TaskGroups))
```

Not sure if it matters much 🤷